artemis/main.py

```diff
@@ -17,7 +17,7 @@
 
 # We run it here so that it will get executed even when importing from main,
 # which will happen when running the app via `uvicorn artemis.main:app`
-db.initialize_database()
+db_migration.migrate_and_start_thread()
```

Comment:
There should some check whether migration from mongo has already happened

Reply:
This  call will start migrating all unmigrated rows. It is needed as there might be a race condition - new web is started but old kartons keep writing to MongoDB. These subsequent migrations will happen every 20 seconds and will be very fast if nothing is to be migrated